keylime/cloud_verifier_tornado.py

```diff
@@ -856,62 +856,76 @@ class AllowlistHandler(BaseHandler):
     def head(self) -> None:
         web_util.echo_json_response(self, 400, "Allowlist handler: HEAD Not Implemented")
 
-    def __validate_input(self, method: str) -> Optional[str]:
+    def __validate_input(self, method: str) -> Tuple[bool, Optional[str]]:
```

Comment:
The change here is a bit confusing, but I also don't see a simpler way to make it nicer.